include/deal.II/base/parameter_handler.h

```diff
@@ -1597,6 +1625,19 @@ class ParameterHandler : public Subscriptor
   virtual bool read_input (const std::string &filename,
                            const bool optional = false,
                            const bool write_stripped_file = false);
+  /**
+   * This is a shell function of read_input(std::string, bool, bool) with a
+   * compact argument list.
+   *
+   * This function will set arguments and flags accroding to the provided
+   * ReadInputFlags and call read_input(std::string, bool, bool). The
+   * ReadInputFlags passed to this function only has effect on this time of
+   * reading.
+   *
+   * Return what read_input(std::string, bool, bool) returned.
```

Comment:
Please change the documentation to be independent of the other ```read_input``` function (which we might deprecate and remove at some point).